src/styles-css/blocks-css/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  max-width: 200px;
+  background: var(--bg);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  margin: 16px auto 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 40px 0 0;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: var(--title-line-hight);
+  color: var(--primary);
+}
+
+.card__product-code {
+  margin-top: 4px;
+}
+
+.product-code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--subtitle-color);
+}
+
+.card__product-rating {
+  margin: 16px 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+}
+
+.product-rating__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--primary);
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: var(--title-line-hight);
+  align-items: flex-end;
+  color: var(--subtitle-color);
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: var(--title-line-hight);
+  color: var(--primary);
+}
+
+.card__button {
+  margin-top: 16px;
+}
+
+.button {
+  text-decoration: none;
+  background-color: var(--button-color);
+  color: var(--bg);
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  border: 1px solid var(--button-color);
+  border-radius: 5px;
+  padding: 11px 70px;
+  text-transform: uppercase;
+}
+
+.button:hover {
+  background-color: transparent;
+  color: var(--button-color);
+}
```

Comment:
We can't use names without underscores for elements. Just use for example card__title and if you want to add margins or positions, so you can use modifier: card__title--(and here your changes).
So, i mean that you should rename your classes and do everything that i wrote above ;)